setup.cfg

```diff
@@ -61,3 +61,42 @@ known_third_party = sphinx, pytest, six, nose, numpy, attr
 known_first_party = astroid
 include_trailing_comma = True
 skip_glob = tests/testdata
+
+[mypy]
+scripts_are_modules = True
+
+[mypy-setuptools]
+ignore_missing_imports = True
+
+[mypy-typed_ast]
+ignore_missing_imports = True
+
+[mypy-dateutil]
+ignore_missing_imports = True
```

Comment:
Not necessary with the `types-python-dateutil` package.

Reply:
Back to 930+ errors 😄